paddle/fluid/framework/details/op_handle_base.h

```diff
@@ -31,6 +33,17 @@ constexpr char kLocalExecScopeName[] = "@LOCAL_SCOPE@";
 // It's responsible for populating necessary fields of ir::Node.
 class OpHandleBase {
  public:
+  /**
+   * NOTE(zjl): Some op should have higher priority than others.
+   * The higher priority op would run first without switching
+   * threads in FastThreadedSSAGraphExecutor.
```

Comment:
The `OpHandle` should not be bound with a specified Executor. So the note should be `The higher priority op should be run first without switching threads in Executor.`

Reply:
Done.